src/components/base/ResponsiveButton.vue

```diff
@@ -0,0 +1,91 @@
+<template>
+	<div>
+		<mq-layout mq="tablet+">
+			<base-button :design="design" v-on="$listeners">
```

Comment:
```suggestion
			<base-button v-on="$listeners" v-bind="$props">
```

Hier v-bind mit $props einfügen, dann werden die props durchgereicht an den baseButton (zB. design, icon)
mit `v-bind="{ ...$attrs, ...$props }"` würdest du alle attribute + definierte props durchreichen